REFERENCE.md

```diff
@@ -35,7 +35,7 @@
 
 rabbitmq
 
- @param $loopback_users. default defined in param.pp. This option configures a list of users to allow access via the loopback interfaces
+ @param loopback_users This option configures a list of users to allow access via the loopback interfaces
```

Comment:
Looks like it still doesn't recognize it as a parameter. Not sure why.

Reply:
I see that. Not sure why, but still get the error
```
[warn]: Missing @param tag for parameter 'loopback_users' near manifests/init.pp:192.
```

Reply:
You could try adding a line with just a `#` after is and before the class definition.

Reply:
I tried that, didn't seem to make a difference.

Reply:
An empty line with no hash seems to work... I _think_ that's right? The diffs are hard to review now.